rdmd.d

```diff
@@ -466,22 +458,20 @@ private int rebuild(string root, string fullExe,
         fullExe = fullExe.defaultExtension(".exe");
 
     // Delete the old executable before we start building.
-    yap("stat ", fullExe);
     if (exists(fullExe))
     {
         enforce(!isDir(fullExe), fullExe ~ " is a directory");
         yap("rm ", fullExe);
         if (!dryRun)
         {
             try
-                   remove(fullExe);
-            catch (FileException e)
+                   std.file.remove(fullExe);
```

Comment:
Why use `std.file.remove` here (with a manual `yap` above) rather than a call to the `remove` wrapper function?

Reply:
Look at the code and I think you'll see why.